ios/Podfile.lock

```diff
@@ -638,4 +638,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ca4eb736278d4211ffaa5cef62299ef8fc2137ca
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.11.3
```

Comment:
Revert

Reply:
@f1sh1918

Reply:
sorry for that. Very annoying...created a ticket for that and reverted podfile in last commit
https://issues.tuerantuer.org/browse/LUN-437